nucypher/control/controllers.py

```diff
@@ -310,14 +310,25 @@ def handle_request(self, method_name, control_request, *args, **kwargs) -> Respo
                            self.emitter.MethodNotFound)
 
         try:
+            # handle request headers
+            request_headers = control_request.headers
+            request_headers = {header.upper(): value for (header, value) in request_headers.items()}
+
             request_data = control_request.data
-            request_body = json.loads(request_data) if request_data else dict()
+            # handle bytes in request body
+            if request_headers.get('CONTENT-TYPE') == 'application/octet-stream':
+                request_body = {"data": bytes(request_data)}
+            # handle JSON in request body
+            else:
+                request_body = json.loads(request_data) if request_data else dict()
 
             # handle query string parameters
             if hasattr(control_request, 'args'):
                 request_body.update(control_request.args)
 
             request_body.update(kwargs)
+            if request_headers.get('X-PROXY-DESTINATION'):
+                request_body['proxy_destination'] = request_headers['X-PROXY-DESTINATION']
```

Comment:
Hmm it's unclear what is meant here, is `proxy_destination` an alias for `X_PROXY_DESTINATION`?